### PR TITLE
[perf] prod version of slurp

### PR DESCRIPTION
**Pull request by rasom**
solves #8381
solves #8383

review only the last commit

All resources loaded by slurp are moved to status-modules/resources dir
in release builds and are loaded only by demand instead of being bundled into
index.*.js.

status: ready

### CODE REVIEW

File: src/status_im/ui/screens/browser/views.cljs [hunk 1]
```diff
@@ -24,8 +24,14 @@
    [status-im.utils.slurp :refer [slurp]]
    [status-im.utils.views :as views]))
 
+(def browser-config-edn
+  (slurp "./src/status_im/utils/browser_config.edn"))
+
 (def browser-config
-  (edn/read-string (slurp "./src/status_im/utils/browser_config.edn")))
+  (let [config (atom nil)]
```
Comment (flexsurfer) [inline src/status_im/ui/screens/browser/views.cljs]: could elaborate on atom? what the purpose?

**Review by flexsurfer — APPROVED** [review]

File: src/status_im/ui/screens/browser/views.cljs (hunk 1 above)
Comment (rasom) [reply src/status_im/ui/screens/browser/views.cljs]: caching of the result, probably simpler to wrap with `memoize`

Comment (rasom) [reply src/status_im/ui/screens/browser/views.cljs]: does it look better now?

Comment (flexsurfer) [reply src/status_im/ui/screens/browser/views.cljs]: it's not about look. i just didn't notice that it's `def` not `defn`, but this looks better :)

**Review by mandrigin — APPROVED** [review]

**Comment by Serhy** [comment]
No regressions in DApp and browsing on iOS and Android (8th, 6.0.1 tested).
Also, can install and use extensions fine (gfycat, wallet 'hello world' extension), can revoke access to Dapp from Profile

**Pull request merged**